test/block-test.js

```diff
@@ -102,6 +105,12 @@ describe('Block', function() {
     assert.bufferEqual(block2.merkleRoot, block2.createMerkleRoot());
   });
 
+  it('should inspect', () => {
+    const [rawBlock] = block482683.getBlock();
```

Comment:
nit: rawBlock would be the `Buffer` (e.g. `Block.fromRaw(raw)`)

Reply:
I think it is ultimately a `fromRaw()`: https://github.com/bcoin-org/bcoin/blob/master/test/util/common.js#L180

But should I change to `block`? I guess that is how the other tests look